control-plane/gateways/metadata_test.go

```diff
@@ -259,6 +259,52 @@ func TestNewMeshGatewayBuilder_Labels(t *testing.T) {
 	}
 }
 
+// The LogLevel for deployment containers may be set on the Gateway Class Config or the Gateway Config.
+// If it is set on both, the Gateway Config takes precedence.
+func TestMeshGatewayBuilder_LogLevel(t *testing.T) {
+	expected := "debug"
```

Comment:
minor, but can we name this variable `debug` or something like that, it's a little difficult to parse it being called `expected` and being passed into the test cases